src/app/shared/auth-guard.service.ts

```diff
@@ -40,13 +40,14 @@ export class AuthService {
 
   // For login route will redirect to main app if there is an active session
   canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> {
-    return this.couchService.get('_session', { withCredentials: true }).pipe(map(res => {
-      if (res.userCtx.name) {
-        this.router.navigate([ '' ]);
-        return false;
-      }
-      return true;
-    }));
+    if (route.url.toString() === 'login') {
+      return this.couchService.get('_session', { withCredentials: true }).pipe(map(res => {
+        if (res.userCtx.name) {
+          this.router.navigate([ '' ]);
+          return false;
+        }
+        return true;
+      }));
+    }
   }
-

```

Comment:
Please put this space back in, there should be no changes to this file in the PR.